### ISSUE TITLE
`'config` in a directory in `~/.config`

### ISSUE THREAD

**Issue by RayZ0rr** [miracle-wm-org/miracle-wm#264]
Maybe I'm overthinking but I think it's better if the default config file is something like `~/.config/miracle/init.yml`.

This will be helpful in the future where:
1. the user can split the config into multiple files and import them using relative path to `~/.config/miracle`
2. the user can add additional scripts at `~/.config/miracle/scripts`
3. the user can add additional plugins at `~/.config/miracle/plugins`

Like I said above these are assumptions for the future.

**Comment by mattkae** [comment]
Yes I'm starting to think this as well. This seems to be the way that 99% of semi-complicated projects are going. I agree that we should do this

**Comment by mattkae** [comment]
I moved it to this directory for v0.6.0 :) `~/.config/miracle-wm/config.yaml`
